src/output_gstreamer.c

```diff
@@ -577,12 +585,48 @@ static int output_gstreamer_init(void)
 	return 0;
 }
 
+static const char *output_gstreamer_version(char *buffer, size_t len)
```

Comment:
this version string also needs to make it all the way up to the `init_logging()` and `do_show_version()` in main. So some more plumbing is needed for that.